### PR TITLE
node test workflow

### PR DESCRIPTION
**Pull request by Intizar-T**
Notion ticket: https://www.notion.so/tokamak/Unit-tests-and-workflow-for-running-the-node-on-every-change-15ed96a400a380c79083c8134c1cac0f?pvs=4

This PR implements:
- node run test which tries running the entire app
- workflow for running the node
- move away from using `cfg` db credentials to `.env` since `cfg` is a versioned file and everyone keeps updating it locally

### CODE REVIEW

File: sequencer/cfg.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ UserWrite     = "postgres"
 ## Password of the PostgreSQL write server
 PasswordWrite = "123"
 ## Name of the PostgreSQL write server database
-NameWrite     = "postgres"
+NameWrite     = "statedb"
```
Comment (Intizar-T) [inline sequencer/cfg.toml:48]: this actually doesn't matter, db initialization no longer depends on `cfg`

File: sequencer/cfg.toml [hunk 2]
```diff
@@ -76,7 +76,7 @@ Rollup   = "0x3d4810E4F8377a34781A6Ceaab9dAc08E361192d"
 
 [Coordinator]
 ## Ethereum address that the coordinator is using to forge batches
-ForgerAddress = "0x7ebd0f4de5e4139b1c07fc4acfa4422c3f4347ed"
+ForgerAddress = "0x46732b099639ff3d3ceeb4b6138ccbbb9b72f492"
```
Comment (Intizar-T) [inline sequencer/cfg.toml:79]: this should be moved to `.env` as well as everyone will accidentally keep pushing their own account address

File: sequencer/database/utils.go [hunk 3]
```diff
@@ -26,17 +25,42 @@ var migrations *migrate.PackrMigrationSource
...
+	if host == "" {
+		host = "localhost"
+	}
+	port, _ = strconv.Atoi(os.Getenv("PGPORT"))
+	if port == 0 {
+		port = 5432
+	}
+	user = os.Getenv("PGUSER")
+	if user == "" {
+		user = "hermez"
```
Comment (Shailu-s) [inline sequencer/database/utils.go]: Why are we using hermez name for the user here ?

Comment (Intizar-T) [reply sequencer/database/utils.go]: right, i missed that, it's a copy paste, will update

**Review by Shailu-s — APPROVED** [review]
Looks good to me.

**Pull request merged**